plugins/package-managers/spdx/src/main/kotlin/utils/SpdxResolvedDocument.kt

```diff
@@ -133,6 +133,16 @@ internal data class SpdxResolvedDocument(
         return pkg
     }
 
+    /**
+     * Retrieve the issues from [issuesByReferenceId] that are not associated with any package [packagesById]. These
+     * issues can be related to general issues with the SPDX document.
+     */
+    fun getIssuesWithoutSpdxPackage(): List<Issue> {
+        return issuesByReferenceId.filterKeys { referenceId ->
```

Comment:
Consider a one-liner expression function like

    issuesByReferenceId.mapNotNull { (id, issue) -> if (packagesById[id] == null) issue else null }

to avoid the final list conversion.